.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,16 @@
+---
+name: Question
+about: Ask a question about the tool.
+title: ''
+labels: 'question'
+assignees: ''
+
+---
+
+<!-- **IMPORTANT!**
+Please make sure to look for an answer to your question in our documentation and the documentation before asking a question here.
+
+If you have a general question regarding the tool use Discord `nuxt` channel. Thanks!
+
+Nuxt Discord: https://discord.nuxtjs.org/
+-->
```

Comment:
I would omit this file in favour of adding a link to GitHub Discussions next to the Discord link in `.github/ISSUE_TEMPLATE/config.yml`

Reply:
@danielroe 

Thank you for feedback!

But, I can't find GitHub Discussions menu in this repository.
It seems to have to be enabled via [Settings](https://docs.github.com/en/discussions/quickstart) of the repository.

I will apply the change you suggested once it appears. 🙋‍♂️

Reply:
Discussions has been enabled.